frontend/packages/console-app/src/components/cloud-shell/cloud-shell-utils.ts

```diff
@@ -98,6 +99,7 @@ export const newCloudShellWorkSpace = (
     },
     annotations: {
       [CLOUD_SHELL_RESTRICTED_ANNOTATION]: 'true',
+      [CLOUD_SHELL_SOURCE_ANNOTATION]: 'web-terminal',
```

Comment:
it's going to appear for new instances, but maybe we also should care about existing ones. WDYT?

Reply:
I don't think there's any mechanism in place to update Web Terminals after creation, and I don't think it's worth adding for this (especially since 4.8 might result in clean installs of WTO).

Reply:
4.8 already happened. 
But yeah, we should be good without updating, if we backport it to 4.8 before WTO 1.3